packages/@adobe/spectrum-css-temp/components/tags/skin.css

```diff
@@ -71,7 +80,8 @@ governing permissions and limitations under the License.
     }
 
     .spectrum-Tags-itemIcon,
-    .spectrum-Tags-itemClearButton {
+    .spectrum-Tags-itemClearButton,
+    .spectrum-Tag-removeButton {
```

Comment:
I'm not sure this is right, what even is the itemClearButton? I can't find an actual use of it which makes me think we just forgot to update it to the new class name

Reply:
I've updated the classNames to those that we actually use.